app/src/main/java/com/example/kakaowebtoon/presentation/ui/home/component/HomeBanner.kt

```diff
@@ -0,0 +1,103 @@
+package com.example.kakaowebtoon.presentation.ui.home.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.kakaowebtoon.R
+import com.example.kakaowebtoon.presentation.util.noRippleClickable
+import com.example.kakaowebtoon.ui.theme.KakaoWebtoonTheme
+
+@Composable
+fun HomeBanner(
+    onBannerClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        modifier = modifier
+            .padding(top = 4.dp)
+            .padding(horizontal = 10.dp),
+        contentAlignment = Alignment.BottomCenter
+    ) {
+        Image(
+            modifier = Modifier.fillMaxWidth(),
+            painter = painterResource(R.drawable.img_home_banner_background),
+            contentDescription = stringResource(R.string.home_banner),
+            contentScale = ContentScale.FillWidth
+        )
+        Image(
+            painter = painterResource(R.drawable.img_home_banner_character),
+            contentDescription = stringResource(R.string.home_banner)
+        )
+        HomeBannerButton(
+            // TODO: 추후 Banner 관련 기능 디벨롭 시 구현
+            modifier = Modifier.noRippleClickable(onBannerClick)
+        )
+    }
+}
+
+@Composable
+private fun HomeBannerButton(
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier
+            .clip(RoundedCornerShape(0.dp, 0.dp, 12.dp, 12.dp))
```

Comment:
```suggestion
            .clip(RoundedCornerShape(topStart = 0.dp, topEnd =  0.dp, bottomEnd = 12.dp, bottomStart = 12.dp))
```
name argument 사용해주시면 가독성 측면에서 더 좋을 것 같습니다!

Reply:
반영하겠습니다~!!!